.editorconfig

```diff
@@ -1,4 +1,3 @@
-# http://editorconfig.org
```

Comment:
why remove this?

Reply:
Seemed like superflous advertising.

Reply:
https://editorconfig.org/ <- we need our logo on it too ;)

@silverwind "... advertising ..." - OK I'm fine with it. you can find this by searching with keyword "editorconfig" too ...